spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -0,0 +1,84 @@
+from typing import Iterable, Union
+import numpy as np
+from scipy.stats import norm
+from spikeinterface.core import BaseRecording, BaseRecordingSegment, get_chunk_with_margin
+from spikeinterface.core.core_tools import define_function_from_class
+from .basepreprocessor import BasePreprocessor, BasePreprocessorSegment
+
+
+class GaussianBandpassFilterRecording(BasePreprocessor):
+    """
+    Class for performing a bandpass gaussian filtering/smoothing on a recording.
```

Comment:
Could explain more what this function do.
Here the gaussian is on the fourier domain.
The immediate intuition would be that this function could a smoother on time domain.
We should have something that make it not ambiguous for users.

Reply:
If you have a perfect resolution: gaussian in the time domain = gaussian in the fourier domain.
Unfortunately that does not hold true when the resolution isn't high enough (e.g. 6kHz with a sf of 30 kHz).

The idea is really simple: it's a convolution between the time series and a gaussian kernel. It's just implemented in the Fourier domain for faster computing.

What do you think is missing from the documentation?

Reply:
Explaining the concept: normally a gaussian smoothing is a low pass filter, but here you make it a bandpass by subtracting two gaussian kernels in the frequency domain. I agree this should be added to the explanation

Reply:
Is this ok?